docs/how_to/create_rectangle_on_map.rst

```diff
@@ -0,0 +1,89 @@
+.. _sunpy-how-to-create-rectangle-on-map:
+
+************************************
+How to Draw a Rectangle on sunpy map
+************************************
+
+``sunpy`` provides a convenient method called :meth:`~sunpy.map.GenericMap.draw_quadrangle` to draw rectangles on maps.
+In this guide, we will demonstrate four different methods to draw a rectangle on a `sunpy.map.Map`.
+
+Specify corners with a single `~astropy.coordinates.SkyCoord`
+=============================================================
+
+We will use one `~astropy.coordinates.SkyCoord` to represent the two opposite corners.
+
+.. code-block:: python
```

Comment:
So these have not rendered at all in the documentation: 

https://sunpy--7435.org.readthedocs.build/en/7435/how_to/create_rectangle_on_map.html

We will need to fix that.

Reply:
I am sorry but what do you exactly mean by not rendered in docs?

Reply:
There is are figures

Reply:
Oh we need figures? Alrighty will do!
And do we need  4 figures or one containing all like we had in the example, with the legend?

Reply:
Let's start with 4 separate ones and we can slim down after if we need to